docs/plugins/gatsby-theme-flow/gatsby-config.js

```diff
@@ -64,6 +64,7 @@ module.exports = ({
           'solidity': 'source.solidity.security',
           'javascript': 'source.js',
           'shell': 'source.shell',
+          'sh': 'source.shell',
```

Comment:
👍

Reply:
Can we add `protobuf` too? Not sure if `source.protobuf` will just work?

Reply:
Good point! I saw it's missing. I'll add in a separate PR